7.0_Hero_Points.md

```diff
@@ -1,4 +1,13 @@
-# 7.0 Hero Points
+# 7.0 Hero Points and Experience Points
 
-**Hero points (HP)** are a resource that you must carefully allocate. They allow you to improve your **result** by exchanging them for a **bump**. They are also the currency you pay to improve your **abilities** over time.
+During play your PC will earn **experience points (XP)**. **XPs** can be used to improve your character. But during a session, before you add your **XPs** to an **experience point pool** with which you can purchase an **improvement package** you can also **burn** them for a boost in a contest. Burning an **XP** gains you a **bump**. You can also **burn** an **XP** on a **plot edit**. Because only **XPs** gained within the session can be used for a **bump** within that session, we call them **hero points (HP)** . At the end of the session of play, unspent **HP** become **XP**
```

Comment:
I think XP is used in this text instead of HP in some places. Like "burning an XP gains you a bump". Please review.